src/data.test.js

```diff
@@ -48,38 +53,40 @@ test('GET data (no password) should fail', async () => {
 });
 
 test('GET data (no password stored) should fail', async () => {
-  await fse.emptyDir(TESTDIR);
+  await fse.emptyDir(TESTDIR, err => {
+    if (err) return console.error(err)
+  });
+  //this is a hack--need to figure out why password file appears in project root.
+  await fse.remove(`${process.env.FILEDIR}/../passwordHash.txt`, err => {
+    if (err) return console.error(err)
+  });
   await request(app)
     .get('/')
     .auth('', tempPassword)
     .expect(401);
 });
 
-test('POST data file should succeed', async () => {
+test('POST with json data should succeed', async () => {
   await request(app)
     .post('/')
-    .attach('file', Buffer.from('heyheyhey'), 'data.txt')
-    .expect(200);
-});
-
-test('POST data with no file should fail', async () => {
-  await request(app)
-    .post('/')
-    .expect(400);
+    .set('Content-Type', 'application/json')
+    .send({ 'data': "sandboxNEU", 'configID': 1 })
+    .expect(200, 'Data saved');
 });
 
-test('POST data with wrong field name should fail', async () => {
+test('POST data with no data should fail', async (done) => {
   await request(app)
     .post('/')
-    .attach('WRONG', Buffer.from('heyheyhey'))
     .expect(400);
+  done()
 });
 
-test('POST data with no file name should fail', async () => {
+test('POST a data file should fail', async (done) => {
   await request(app)
     .post('/')
-    .attach('file', Buffer.from('heyheyhey'))
+    .attach('file', Buffer.from('heyheyhey'), 'data.txt')
     .expect(400);
+  done()
 });
 
```

Comment:
Can we get some tests to verify the contents returned by the GET request contain what we POSTed? Since it's JSON now, we can actually write tests on it.